### PR TITLE
CI-905 indicate apprenticeship type

### PR DESCRIPTION
**Pull request by froberg**
Table under apprenticeships on estimated cost page is too wide. 
Waiting for Rajesh to make a decision on what to do.
/c
--
Suggested solution will not work
Unfortunately we cannot any Jas solutions on our applications due to accessibility’s standards. (Rajesh)
/c

### CODE REVIEW

File: src/SFA.DAS.Forecasting.Web/ViewModels/EstimationApprenticeshipViewModel.cs [hunk 1]
```diff
@@ -7,14 +7,26 @@ namespace SFA.DAS.Forecasting.Web.ViewModels
...
+        public FundingSource FundingSource { get; set; }
+
+        public string FundingSourceText
+        {
+            get
+            {
+                if (FundingSource == 0 || FundingSource == FundingSource.Transfer)
+                    return "Employer transfer";
+                if(FundingSource == FundingSource.Levy)
+                    return "Account Funds";
```
Comment (dashton82) [inline src/SFA.DAS.Forecasting.Web/ViewModels/EstimationApprenticeshipViewModel.cs]: might need to be lowercase F

File: src/SFA.DAS.Forecasting.Domain/Estimations/AccountEstimationProjectionRepository.cs [hunk 2]
```diff
@@ -34,12 +35,19 @@ public class AccountEstimationProjectionRepository : IAccountEstimationProjectio
...
 
-            var employerCommitmentsModel = new EmployerCommitmentsModel();
-            employerCommitmentsModel.SendingEmployerTransferCommitments = commitments;
+            var employerCommitmentsModel = new EmployerCommitmentsModel
+            {
+                SendingEmployerTransferCommitments = commitments
+                    .Where(m => m.FundingSource == Models.Payments.FundingSource.Transfer || m.FundingSource == 0)
+                    .ToList(),
+                LevyFundedCommitments = commitments
+                    .Where(m => m.FundingSource == Models.Payments.FundingSource.Levy)
```
Comment (dashton82) [inline src/SFA.DAS.Forecasting.Domain/Estimations/AccountEstimationProjectionRepository.cs]: Might be worth changing this to be anything other than funding source Transfer - as part of the work coming up is to bring in some of the other payment types for when the payment is co-investment

Comment (froberg) [reply src/SFA.DAS.Forecasting.Domain/Estimations/AccountEstimationProjectionRepository.cs]: Don't we want to be more explicit about what goes in to that list?
If I change it, would it still be **LevyFundedCommitments** then?

Comment (dashton82) [reply src/SFA.DAS.Forecasting.Domain/Estimations/AccountEstimationProjectionRepository.cs]: Fair point, lets leave this for now as is 😄

**Pull request merged**